undertow/src/main/java/org/wildfly/swarm/undertow/UndertowFraction.java

```diff
@@ -53,14 +53,14 @@ public void postConstruct() {
 
```

Comment:
No changes, just formatting from IntelliJ IDEA. 
Could you do a little test by formatting this class in your IDE to check if it changes the indentation too? I think my IDE might be configured differently from the others.